src/hotspot/share/jeandle/jeandleAbstractInterpreter.cpp

```diff
@@ -1564,7 +1564,7 @@ void JeandleAbstractInterpreter::do_array_store_inner(BasicType basic_type, llvm
 }
 
 void JeandleAbstractInterpreter::do_array_store(Bytecodes::Code code) {
-  llvm::Value* value = nullptr;
+  llvm::Value *value = nullptr;
```

Comment:
```suggestion
  llvm::Value* value = nullptr;
```
Don't change unrelated code.